packages/common/src/featureFlag.js

```diff
@@ -3,9 +3,14 @@ const isFeatureActive = async (featureFlagName, localPlanningAuthorityCode) => {
 		return true;
 	}
 
-	const { AppConfigurationClient } = require('@azure/app-configuration');
+	//if no env variable pointing to the config in azure, early return to avoid issues.
 	const config = require('./config');
-	const logger = require('../lib/logger');
+	if (config.featureFlagging.endpoint === undefined) {
```

Comment:
Just wondering, are there any cases where the endpoint could be invalid that might be missed by this check? For example if it's an empty string or null.